api/models/team.py

```diff
@@ -0,0 +1,29 @@
+"""Import Django models."""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
```

Comment:
That functionality is mostly embedded in Python itself. Underscore is indeed used as convention for translations in Python.